packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -65,6 +70,17 @@ type State = {|
  * the `modal` prop.
  */
 export default class ModalLauncher extends React.Component<Props, State> {
+    static getDerivedStateFromProps(props: Props, state: State) {
+        return {
+            opened:
+                typeof props.opened === "boolean" ? props.opened : state.opened,
```

Comment:
Doesn't this mean that if any prop changes, the modal might suddenly reappear?

Reply:
The `opened` and `children` props shouldn't be used together.  I should probably throw an error if we detect that situation and provide a helpful warning message.  I'm not sure how changing any prop would cause the modal to reappear.  Can you elaborate?

Reply:
Well, `getDerivedStateFromProps` gets called whenever props change, so this would then set the opened state back to `true` when props change.

Reply:
We discussed this and decided it should be fine except in the case where devs pass both children and opened so I'm going to add an exception in that case.